environment/src/main/java/com/sequenceiq/environment/configuration/RemoteTokenConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.sequenceiq.environment.configuration;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.oauth2.provider.token.ResourceServerTokenServices;
+
+import com.sequenceiq.cloudbreak.auth.altus.GrpcUmsClient;
+import com.sequenceiq.cloudbreak.auth.caas.CaasClient;
+import com.sequenceiq.cloudbreak.auth.uaa.IdentityClient;
+import com.sequenceiq.cloudbreak.common.service.token.CachedRemoteTokenService;
+
+@Configuration
+public class RemoteTokenConfig {
+
+    @Value("${environment.client.id}")
+    private String clientId;
+
+    @Value("${environment.client.secret}")
+    private String clientSecret;
+
+    @Inject
+    @Named("identityServerUrl")
+    private String identityServerUrl;
+
+    @Inject
+    private IdentityClient identityClient;
+
+    @Inject
+    private GrpcUmsClient umsClient;
+
+    @Inject
+    private CaasClient caasClient;
+
+    @Bean
+    public ResourceServerTokenServices remoteTokenServices() {
```

Comment:
We already have a `ResourceServerTokenServices` bean from here: 
`com.sequenceiq.cloudbreak.auth.security.token.CommonRemoteTokenConfig`